packages/notifi-frontend-client/lib/models/SubscriptionCardConfig.ts

```diff
@@ -152,6 +165,7 @@ export type EventTypeItem =
   | BroadcastEventTypeItem
   | HealthCheckEventTypeItem
   | TradingPairEventTypeItem
+  | FusionMultiThreshholdEventTypeItem
```

Comment:
Here as mentioned above, we will not need to have a new type defined here. We can just reuse common `FusionEventTypeItem`